### PR TITLE
Add VidExpression

### PR DESCRIPTION
**Pull request by laura-ding**

### CODE REVIEW

File: src/common/expression/EdgeExpression.h [hunk 1]
```diff
@@ -30,7 +36,7 @@ class EdgeExpression final : public Expression {
     }
 
     std::string toString() const override {
-        return "EDGE";
```
Comment (Shylock-Hg) [inline src/common/expression/EdgeExpression.h:33]: Why don't return 'edge' directly?

File: src/common/expression/VertexExpression.h [hunk 2]
```diff
@@ -30,7 +36,7 @@ class VertexExpression final : public Expression {
     }
 
     std::string toString() const override {
-        return "VERTEX";
```
Comment (Shylock-Hg) [inline src/common/expression/VertexExpression.h:33]: ditto

File: src/common/expression/EdgeExpression.h [hunk 3]
```diff
@@ -46,6 +52,7 @@ class EdgeExpression final : public Expression {
 
 private:
     Value                                   result_;
+    std::unique_ptr<std::string>            outPutName_;
```
Comment (monadbobo) [inline src/common/expression/EdgeExpression.h:55]: maybe "name_" was better.

File: src/common/expression/EdgeExpression.h (hunk 1 above)
Comment (laura-ding) [reply src/common/expression/EdgeExpression.h:33]: Because  the user can set the return name without aliai, just from the label

Comment (Shylock-Hg) [reply src/common/expression/EdgeExpression.h:33]: In fact, better to change the return name by alias instead of change the expression.

**Pull request closed without merging**